static/js/place/subtitle.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+
+interface SubtitlePropsType {
+  category: string;
+  dcid: string;
+}
+
+class Subtitle extends React.Component<SubtitlePropsType, unknown> {
```

Comment:
please add a component description (and can we call this PageSubtitle to be more specific?)

Reply:
Done